immutablecollections/_immutableset.py

```diff
@@ -235,6 +236,12 @@ def __sub__(self, other: AbstractSet[Any]) -> "ImmutableSet[T]":
         """
         return self.difference(other)
 
+    def __rsub__(self, other: AbstractSet[V]) -> "AbstractSet[V]":
+        """
+        Subtracts this set from a different set
```

Comment:
Can you instead provide some motivation of why we need `__rsub__`? You could say the same thing about normal `__sub__`.

Reply:
Nice docstring 👍